javascript/node/selenium-webdriver/lib/webdriver.js

```diff
@@ -2021,9 +2020,8 @@ class WebElement {
    *     have been typed.
    */
   async sendKeys(...args) {
-    let keys = [];
-    (await Promise.all(args)).forEach(key => {
-      let type = typeof key;
+    let keys = (await Promise.all(args)).reduce((splitedKeys, key) => {
```

Comment:
This is not going to work, it breaks the complete `sendKeys` method implementation. i would suggest to revert it to original!